calblueprint/meepanyar
# Financial Summary Screens

## 🖋 PR Description

This PR adds in the financial summary screens as screenshotted below, and also pulls in data from Airtable to populate the fields. Currently a specific financial report is being pulled instead of the one associated with a site since I'm not quite sure how to go about that yet (does it need some sort of context so that we know which site we're pulling from? etc)

### 🤳Screenshots (if applicable)

![screen1](https://user-images.githubusercontent.com/35015640/99048056-ecc36200-2549-11eb-976f-2933e6d18e1d.PNG)
![screen2](https://user-images.githubusercontent.com/35015640/99048058-ee8d2580-2549-11eb-9645-6583ae3dd348.PNG)

### 🐞 Bug Report
![warnings](https://user-images.githubusercontent.com/35015640/99048945-17fa8100-254b-11eb-81fa-8c40eb1b55ba.PNG)
Currently running into these warnings right now and I don't know enough about refs to understand what it's talking about :') But despite the warnings everything shows up properly. Also I'm not quite sure how your profit and Mee Panyar's profit is calculated based off of the total profit, so I just input manual zeroes in for now but that can be changed once that's figured out.

### 🧪 Testing

How do we test your feature?
I currently redirected the Incidents page to this screen since I don't think the link from the home page to this is available yet? This will have to be changed later on, but you can go to http://localhost:3000/financialsummary after running npm start to check these files out! Also I can't get Helvetica Neue on my computer so I tested with Arial so if someone would be able to test with the right font for me that would be wonderful :')

cc: @jenmhoang
